content/60.methods.md

```diff
@@ -187,148 +181,151 @@ Manubot users can write in the simple markdown format, render the manuscript in
+Manuscripts are written in markdown and can be rendered in several formats providing different advantages to users.
 For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, DOCX, and now, LaTeX (in a more limited capacity).
-The interactive HTML manuscript format offers several advantages over a static PDF to harmonize available resources that we were able to apply to specific problems of COVID-19.
-The integration of scite into the HTML build makes references more manageable by visually representing whether their results are contested or whether they have been corrected or retracted.
+The interactive HTML manuscript format offers several advantages over a static PDF to harmonize available resources and address specific problems related to COVID-19.
+The integration of scite into the HTML build makes references more manageable by visually indicating whether their results are contested or whether they have been corrected or retracted.
 Cross-referencing different pieces of the manuscript, such as cited preprints with reviews stored in an appendix, is another interactive option presented by HTML.
-The DOCX format was preferred by most non-technical users for reviewing the final version of the manuscript and was useful for creating submissions to biological journals.
+The DOCX format was preferred by most non-technical users for reviewing the final version of the manuscript and was useful for creating submissions to a biological journal.
 Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate DOCX outputs was expanded to allow users to generate DOCX files containing only a section of the manuscript.
 In our case, where the full project is nearly 150,000 words, this allows individual pieces to be shared more easily.
 Finally, the preliminary addition of LaTeX output is useful for researchers from computational fields who submit papers in TeX format and removes the step of reformatting markdown prior to submission. 
 
-| Additions to Manubot | 
-|:-----------------|
-| CI scripts to regularly download external data sources, generate figures and statistics, and store them in a GitHub branch that is read when Manubot builds the latest manuscript |
-| Zotero extension to report more relevant clinical trial metadata from <https://clinicaltrials.gov> |
-| Manubot feature to support citing any Compact Uniform Resource Identifier, such as clinicaltrials or ncbigene |
-| Integration with scite to track retractions, corrections, and notices of concern |
-| CI spell-checking reports that post as pull request comments |
-| Improved support for Pandoc's LaTeX output |
-| Building individual sections of a larger manuscript as standalone documents alongside the complete manuscript |
-
-Table: List of extensions added to Manubot's functionalities for the COVID-19 collaborative review.
+| Type | Description |
```

Comment:
I like the 2-column format!

Reply:
Yeah, the 1 column table didn't look good in the LaTeX PDF.  This made it look more like an actual table and saved a little space too.